quant/nncf/qat/objectdetection.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, model, loaders=None, **kwargs):
         self.factor = kwargs.get("LR_SCHEDULER_FACTOR", 1)
         self.val_interval = kwargs.get("VALIDATION_INTERVAL", 1)
         self.weight_decay = kwargs.get("WEIGHT_DECAY", 0.0005)
-
+        self.work_path = os.getcwd()
```

Comment:
os.getcwd() is not fixed and tends to change in between operations. would be good if we do something like 
os.dirname(__file__) to get the true folder. 

also, the following may work exactly as expected.
```suggestion
        self.work_path = ROOT # from nyuntam.settings import ROOT
```